### PR TITLE
Build documentation site in docs directory

### PR DESCRIPTION
**Pull request by ChrisMacNaughton**

### CODE REVIEW

**Comment by ChrisMacNaughton** [comment]
https://canonical-microceph--86.com.readthedocs.build/en/86/ has a demo of this site building!

File: docs/tutorial/initial_setup.rst [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    lxc exec microceph-3 microceph cluster join eyJuYW1lIjoibWljcm9jZXBoLTMiLCJzZWNyZXQiOiIyOWJhZGExZGYzYTMyYjJiNDRiZDIwYTliM2QwNGRiNzBjMjYzMTZlZjZmYjkzYTJhOTVkYjgzMWEwMmFjNGYwIiwiZmluZ2VycHJpbnQiOiJkMjRkOGJiYjY0MTgyMWFlMjhkY2VlYWM2YmNkMGU4MmY1M2U2OTdmNDJjM2EyZTc0ZjhkMTk4MDhmNzZiNjgyIiwiam9pbl9hZGRyZXNzZXMiOlsiMTAuNjIuNzMuMTE0Ojc0NDMiXX0=
+
+You should now have a Ceph monitor cluster with quorum:
+
+.. code-block:: shell
+
+    $ lxc exec microceph-1 microceph.ceph status
+    cluster:
+        id:     a95eaf13-c3fe-466a-8278-d92186effbec
+        health: HEALTH_WARN
```
Comment (lmlg) [inline docs/tutorial/initial_setup.rst:32]: I think reassuring readers that the `HEALTH_WARN` message is to be expected for now and that it should disappear once the OSD's are up.

File: docs/tutorial/install.rst [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    lxc launch images:ubuntu/22.04/cloud microceph-3 --vm
+
+Add storage
+~~~~~~~~~~~
+
+.. code-block:: shell
+
+    for i in $(seq 1 3); do
+        lxc storage volume create default osd-$i --type block size=10GiB
+        lxc config device add microceph-$i osd-$i disk pool=default source=osd-$i
```
Comment (lmlg) [inline docs/tutorial/install.rst:21]: Does this need any previous configuration setting previously set for it to work? I seem to recall that it did, but I may be wrong.

**Review by lmlg — COMMENTED** [review]
Maybe a troubleshooting section as well? I know this is all very green, but pointing to where logs are could be useful for the time being.

**Comment by ChrisMacNaughton** [comment]
Given that documentation completeness isn't actually a goal of this PR, but rather to get us having some docs, somewhere, that we can contribute to, I'd suggest that your suggestions could be added independently of this change!

**Comment by ChrisMacNaughton** [comment]
I've captured some of the comments here in https://github.com/canonical/microceph/issues/90, https://github.com/canonical/microceph/issues/91, and https://github.com/canonical/microceph/issues/92

**Pull request merged**